### PR TITLE
Hcms 11 feature create space

### PR DESCRIPTION
**Pull request by jannikr**
- VUE init
- Components (Space)
- Routes
- Bootstrap (we can change that later - for me its faster to apply)
- Server-Communication (axios)

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1 +1 @@
-.idea
```
Comment (rgabisch) [inline .gitignore:1]: please reinsert `.idea`, @jannikr.

File: frontend/public/audity/src/components/Space.vue [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+            { name: 'Scrum-Podcast'},
+            { name: 'Mein Reisepodcast'}
+        ],
+        spaceName: ''
+    }),
+    methods: {
+        addSpace(){
+            console.log('Add:' + this.spaceName);
+            // [JNR] check duplicates
+            axios.put('http://localhost:3000/spaces', {name: this.spaceName, userid: '0'}).then((response) => {
```
Comment (rgabisch) [inline frontend/public/audity/src/components/Space.vue:43]: Based on the ticket [HCMS-9](https://headless-cms.atlassian.net/browse/HCMS-9) a new `Space` is created via a `POST` and not a `PUT`.

Maybe we should change `axios.put(...)` to `axios.post(...)`?

Comment (jannikr) [reply frontend/public/audity/src/components/Space.vue:43]: Sure.

File: .gitignore (hunk 1 above)
Comment (jannikr) [reply .gitignore:1]: do you mean in root-d?

Comment (rgabisch) [reply .gitignore:1]: yes, inside the root directory 👍

**Pull request merged**